doc/man3/d2i_PrivateKey.pod

```diff
@@ -29,6 +29,7 @@ i2d_PrivateKey_fp
                               long length);
 
  int i2d_PrivateKey(const EVP_PKEY *a, unsigned char **pp);
+ int i2d_PKCS8PrivateKey(const EVP_PKEY *a, unsigned char **pp);
```

Comment:
There is an existing page for i2d_PKCS8PrivateKey_* and friends. It might make more sense for this function to be documented there.

Reply:
My reason for putting where I did, is that this has the same signature and purpose as `i2d_PrivateKey()`, which is quite different from the existing `i2d_PKCS8...` functions. This function is narrowly just a way to reliably get unencrypted PKC8 via `i2d`.